examples/js/loaders/GLTFLoader.js

```diff
@@ -2013,7 +2013,7 @@ THREE.GLTFLoader = ( function () {
 
 				var onLoad = resolve;
 
-				if ( useImageBitmap ) {
```

Comment:
can the `useImageBitmap` property be removed as well?

Reply:
Yes, indeed! 🎉